subsys/net/lib/zperf/zperf_shell.c

```diff
@@ -335,10 +466,24 @@ static int cmd_udp_download(const struct shell *sh, size_t argc,
 
 		if (argc >= 2) {
 			param.port = strtoul(argv[1], NULL, 10);
+			if (param.port < 1 || param.port > 65535) {
```

Comment:
`param.port` is uint16_t, so `param.port > 65535` is always false, this will likely be a source of complaint from static analyzer. I'd just check for `!= 0` here.